pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -95,7 +96,8 @@
 
 public class PinotLLCRealtimeSegmentManager {
   public static final Logger LOGGER = LoggerFactory.getLogger(PinotLLCRealtimeSegmentManager.class);
-  private static final String KAFKA_SMALLEST_OFFSET = "smallest";
+  private static final OffsetCriteria SMALLEST_OFFSET_CRITERIA =
```

Comment:
Minor: Do you want to make SMALLEST_OFFSET_CRITERIA a public static in OffsetCriteria class?

Reply:
yes that would be a better place. Will change it